nxdrive/data/qml/TransferItem.qml

```diff
@@ -71,11 +71,10 @@ Rectangle {
                     icon: MdiFont.Icon.close
                     tooltip: qsTr("CANCEL") + tl.tr
                     iconColor: "red"
-                    enabled: !finalizing
+                    property int p : progress || 0.0
+                    enabled: !(status == "CANCELLED" || finalizing)
                     onClicked: {
-                        enabled = false
-                        var confirmed = application.confirm_cancel_transfer(engine, uid, name)
-                        enabled = !(confirmed || finalizing)
+                        application.confirm_cancel_transfer(engine, uid, name)
```

Comment:
You should revert my changes done on `confirm_cancel_transfer()` (c53de92fc7620f7fb850fe2df95a4c7c8f486c0d).